gtclang/src/gtclang/gtc-parse.cpp

```diff
@@ -112,19 +115,21 @@ int main(int argc, char* argv[]) {
 
   // Parse format to enumeration
   dawn::SIRSerializer::Format format;
-  if(result["format"].as<std::string>() == "json")
+  const std::string sirFormat = result["format"].as<std::string>();
+  if(sirFormat == "json")
     format = dawn::SIRSerializer::Format::Json;
-  else if(result["format"].as<std::string>() == "byte")
+  else if(sirFormat == "byte")
     format = dawn::SIRSerializer::Format::Byte;
   else
-    throw std::runtime_error("Unknown SIR format");
+    throw std::runtime_error(std::string("Unknown SIR format: ") + sirFormat +
+                             ". Options are [json | byte]");
```

Comment:
As of our coding conventions, we should use braces also for one-liners

Reply:
Thanks for catching that!